src/parser/tbc/modules/items/EnchantChecker.tsx

```diff
@@ -114,6 +116,7 @@ const MAX_ENCHANT_IDS = [
   2934, // https://tbc.wowhead.com/spell=46512/enchant-gloves-blasting
   1594, // https://tbc.wowhead.com/spell=46511/enchant-gloves-assault
   2564, // https://tbc.wowhead.com/spell=25080/enchant-gloves-superior-agility
+  2613, // https://tbc.wowhead.com/spell=25072/enchant-gloves-threat
```

Comment:
```suggestion
  2613, // https://tbc.wowhead.com/spell=25072/enchant-gloves-threat
  684, // https://tbc.wowhead.com/spell=33995/enchant-gloves-major-strength
```

Reply:
This is one I missed when adding other enchants. It's used by warriors, so probably nice to have here.